tests/MinMapper2.paf

```diff
@@ -72,7 +72,7 @@ datasets: {
         tables:      "raw"
     }
     forced_src: {
-        template:    "%(tract)d/%(filter)s/forced_src-v%(visit)d-c%(ccd)02d.boost"
+        template:    "%(tract)d/%(filter)s/forced_src-v%(visit)d-c%(ccd)02d.fits"
```

Comment:
Not clear to me why this one is switching from boost to fits, while all the others changed from boost to yaml.

Reply:
Ah, glad you caught this; I was going to make it a separate commit but forgot.  Anyhow, this is `.fits` because in this case it always should have been - the file being saved actually is a FITS file, and we've just been using the wrong extension by mistake.

Reply:
I was wondering if that was what happened 😄 a separate commit would be best, please.